schemas/com.snowplowanalytics.telemetry/tracker_telemetry/jsonschema/1-0-0

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for tracker telemetry",
+  "self": {
+    "vendor": "com.snowplowanalytics.telemetry",
+    "name": "tracker_telemetry",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "collectorHost": {
+      "type": "string",
+      "description": "Host URL of the tracker collector, e.g. https://collector.acme.com",
+      "format": "uri"
+    },
+    "collectorPath": {
+      "type": "string",
+      "description": "Path of the tracker collector, e.g. /com.snowplowanalytics.snowplow/tp2"
+    },
+    "appId": {
+      "type": "string",
+      "description": "The Application Identifier of the tracker",
+      "minLength": 1,
+      "maxLength": 1024
+    },
+    "pageUrl": {
+      "type": "string",
+      "description": "Full URL of the page that the tracker is running on",
+      "format": "uri"
+    }
```

Comment:
Can we have optional `trackerNamespace` and `trackerVersion` fields?

Reply:
added